python-package/xgboost/sklearn.py

```diff
@@ -494,10 +509,18 @@ def predict_proba(self, data, output_margin=False, ntree_limit=0):
         class_probs = self.booster().predict(test_dmatrix,
                                              output_margin=output_margin,
                                              ntree_limit=ntree_limit)
+        # Ensure that class_probs are between 0 and 1 (min-max transform)
+        scaled_class_probs = (class_probs - np.min(class_probs, axis=0)) / \
```

Comment:
I'll add a test case and resubmit. I suppose this will have to be in the file https://github.com/dmlc/xgboost/blob/master/tests/python/test_with_sklearn.py